### PR TITLE
enh(deps): Fix Material UI breaking changes and update dependencies 

### PR DESCRIPTION
**Pull request by Thebarda**
https://github.com/mui-org/material-ui/releases

### CODE REVIEW

File: src/ListingPage/index.stories.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable react/require-default-props */
```
Comment (jdelpierre) [inline src/ListingPage/index.stories.tsx:2]: little question : What does that mean?

File: src/InputField/Search/PersistentTooltip.tsx [hunk 2]
```diff
@@ -87,4 +87,10 @@ const PersistentTooltip = ({
   );
 };
 
+PersistentTooltip.defaultProps = {
+  openTooltip: undefined,
+  toggleTooltip: undefined,
+  closeTooltip: undefined,
+};
+
```
Comment (bdauria) [inline src/InputField/Search/PersistentTooltip.tsx:95]: This shouldn't be needed anymore

File: src/InputField/Search/PersistentTooltip.tsx [hunk 3]
```diff
@@ -32,37 +32,33 @@ interface Props {
 const PersistentTooltip = ({
   children,
   labelSearchHelp,
-  openTooltip,
-  toggleTooltip: toggleTooltipProp,
-  closeTooltip: closeTooltipProp,
+  openTooltip = undefined,
+  toggleTooltip = undefined,
+  closeTooltip = undefined,
```
Comment (bdauria) [inline src/InputField/Search/PersistentTooltip.tsx:37]:
```suggestion
  openTooltip,
  toggleTooltip,
  closeTooltip,
```

File: src/InputField/Search/RegexpHelpTooltip.tsx [hunk 4]
```diff
@@ -39,14 +39,14 @@ interface TooltipProps {
 }
 
 const RegexpHelpTooltip = ({
-  description,
+  description = undefined,
```
Comment (bdauria) [inline src/InputField/Search/RegexpHelpTooltip.tsx:42]:
```suggestion
  description,
```

File: src/InputField/Search/RegexpHelpTooltip.tsx [hunk 5]
```diff
@@ -39,14 +39,14 @@ interface TooltipProps {
...
-  labelTips,
-  labelGetHelp,
-  urlTip,
-  labelUrlTip,
+  labelExamples = undefined,
+  examples = undefined,
+  labelTips = undefined,
+  labelGetHelp = undefined,
+  urlTip = undefined,
+  labelUrlTip = undefined,
```
Comment (bdauria) [inline src/InputField/Search/RegexpHelpTooltip.tsx:49]:
```suggestion
  labelExamples,
  examples,
  labelTips,
  labelGetHelp,
  urlTip,
  labelUrlTip,
```

File: src/Listing/Header.tsx [hunk 6]
```diff
@@ -47,8 +47,8 @@ const ListingHeader = React.forwardRef(
   (
     {
       onSelectAllClick,
-      order,
-      orderBy,
+      order = undefined,
+      orderBy = undefined,
```
Comment (bdauria) [inline src/Listing/Header.tsx:51]:
```suggestion
      order,
      orderBy,
```

File: src/ListingPage/index.stories.tsx (hunk 1 above)
Comment (bdauria) [inline src/ListingPage/index.stories.tsx:2]:
```suggestion
```

File: src/Snackbar/index.tsx [hunk 7]
```diff
@@ -36,7 +36,12 @@ interface Props {
   severity: Severity;
 }
 
-const Snackbar = ({ message, open, onClose, severity }: Props): JSX.Element => {
+const Snackbar = ({
+  message,
+  open,
+  onClose = undefined,
```
Comment (bdauria) [inline src/Snackbar/index.tsx:42]:
```suggestion
  onClose,
```

**Comment by Thebarda** [comment]
Closed in favor of #263

**Pull request closed without merging**